easypy/units.py

```diff
@@ -323,6 +324,7 @@ def render(self, unit=None, precision=None):
         ss = float(self)
         if unit not in ("ms", "s"):
             mm, ss = divmod(ss, 60)
+            ss = floor(ss)
```

Comment:
what if the duration is less than a second? flooring is not always the right thing.

Reply:
This logic is only relevant when showing minutes, so operations in the following section is unaffected
```pycon
In [1]: from easypy.units import *

In [2]: 0.5 * SECOND
Out[2]: 500ms

In [3]: 0.9 * SECOND
Out[3]: 900ms

In [4]: 0.99 * SECOND
Out[4]: 990ms

In [5]: 0.999 * SECOND
Out[5]: 999ms

In [6]: 0.9999 * SECOND
Out[6]: 1000ms

In [7]: 1.999 * SECOND
Out[7]: 2.0s

In [8]: 1.99 * SECOND
Out[8]: 2.0s

In [9]: 1.9 * SECOND
Out[9]: 1.9s
```

If `1:59.9` should become `01:59m`, then `1:01.9` should also become `01:00m` (I guess)
```pycon
In [10]: 1 * MINUTE + 0.5 * SECOND
Out[10]: 01:00m

In [11]: 1 * MINUTE + 0.9 * SECOND
Out[11]: 01:00m
```

This section is also unaffected by my changes
```pycon
In [12]: 59.9 * SECOND
Out[12]: 59.9s

In [13]: 59.99 * SECOND
Out[13]: 60.0s
```